entities/playerJump.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseEntity, Column, CreateDateColumn, Entity, ManyToOne, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { PlayerInfo } from './playerInfo.entity';
+import { Round } from './round.entity';
+
+/**
+ * Database record
+ */
+@Entity()
+export class PlayerJump extends BaseEntity {
+  /**
+   * Primary key
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
```

Comment:
Is it possible to get the duration? or the time he landed? (just wondering)

Reply:
Unless the player takes fall damage (:smile: ) I don't know if that'll be possible. Not easily anyways. Would have to check once I start playing around with demos